packages/studio-ui/src/web/components/Layout/TopNav/index.tsx

```diff
@@ -0,0 +1,80 @@
+import { Button, Position } from '@blueprintjs/core'
+import { lang, ToolTip, utils } from 'botpress/shared'
+import cx from 'classnames'
+import React, { FC } from 'react'
+import { RiLayoutLeftLine, RiLayoutRightLine, RiLayoutBottomLine } from 'react-icons/ri'
+import { connect } from 'react-redux'
+import { toggleBottomPanel, toggleExplorer } from '~/actions'
+
+import { RootReducer } from '../../../reducers'
+import EnterPriseTrial from './EnterpriseTrial'
+import RightToolBar from './RightToolbar'
+
+import style from './style.scss'
+
+type Props = ReturnType<typeof mapStateToProps> & typeof mapDispatchToProps
+
+const Toolbar: FC<Props> = props => {
+  // We cannot use setEmulatorOpen as it's actually the state is actually controlled by the message passed from the iframe
+  // Will need some refactor for the emulator
+  // This should be moved this somewhere else, it seems to be all around the app
+  const toggleEmulator = () => {
+    window.botpressWebChat.sendEvent({ type: 'toggle' })
+  }
```

Comment:
This could be moved outside of the component / doesn't need to be created every render